umbrella/Cargo.toml

```diff
@@ -174,18 +173,27 @@ std = [
 	"snowbridge-beacon-primitives?/std",
 	"snowbridge-core?/std",
 	"snowbridge-ethereum?/std",
-	"snowbridge-outbound-queue-merkle-tree?/std",
+	"snowbridge-inbound-queue-primitives?/std",
+	"snowbridge-merkle-tree?/std",
+	"snowbridge-outbound-queue-primitives?/std",
+	"snowbridge-outbound-queue-runtime-api-v2?/std",
```

Comment:
not specific to this PR, but I would remove snowbridge pallets from the `umbrella` crate - there are no other users of these pallets outside the fellowship who don't use umbrella anyway.

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/25